### DISCUSSION TITLE
Feed

### DISCUSSION THREAD

**Discussion by MetinSeven** [OpenModelDB/open-model-database#246] [General]
Great initiative.

I'd love to see an ATOM / RSS feed integrated, so you can keep track of new additions via a feed reader.

Thanks!

**Comment by RunDevelopment** [comment]
Then we have to start tracking when models are added to the database. Currently, we only track when the model was released by the author (on any platform).

This also creates an interesting question: Should the feed be sorted by date of release (any platform) or by when then model was added to the database?

**Reply by MetinSeven** [reply]
> This also creates an interesting question: Should the feed be sorted by date of release (any platform) or by when then model was added to the database?

I'd vote for date of release, as newer models often tend to offer improved results.

Another suggestion: a user rating system for each model (e.g. 0 to 5 stars with half-star increments), with the addition of sorting by rating.
